control-services/.env.example

```diff
@@ -0,0 +1,9 @@
+# The following environment variables are used in the various services
+# Copy the lines below into a ".env" file and change passwords/usernames
+
+gitea_db_user=gitea
+gitea_db_password=gitea
+semaphore_admin_password=cangetin
```

Comment:
These credentials are held in `config.json`.  We don't need to put credentials in another place and cause confusion whenever we change passwords/jump networks.  I dont see how this env file will inform future devs unless your recommending a design change to the existing structure of storing/referencing service credentials?

Reply:
Credentials, if stored in config.json or .env should not be committed to the repo.
Is there any documentation for using `config.json`? I've only ever seen [docker-compose .env](https://docs.docker.com/compose/environment-variables/#the-env-file)

Reply:
Not the most up to date version but heres documentation https://github.com/dev4vater/vater/wiki/Configuration-File